.github/workflows/main.yml

```diff
@@ -7,12 +7,24 @@ on:
     branches: [ master ]
 
 jobs:
+  setup_flutter_versions:
+    runs-on: ubuntu-latest
+    outputs:
+      flutter-versions: ${{ steps.id_flutter_versions.outputs.versions }}
+    steps:
+      - name: Fetch Flutter versions
+        id: id_flutter_versions
+        run: |
+          VERSIONS=$(curl -s https://storage.googleapis.com/flutter_infra_release/releases/releases_linux.json | jq -c '[.releases | map(select(.channel == "stable")) | group_by(.version | split(".")[:2] | join(".")) | map(.[0]) | sort_by(.release_date) | reverse | .[:5] | map(.version)] | flatten + ["stable", "beta"]')
```

Comment:
My bash-fu isn't very strong, so just to double check that this does the following?

Grab the last 4 stable releases, taking the most recent patch version (so for example 3.35.99) and add the current "stable" + "beta" to it?

Can we move this to a script that gets its own unit test?

TIL about https://storage.googleapis.com/flutter_infra_release/releases/releases_linux.json

Also, we now test 6! stable releases, which is way mor than Flutter's own policy of "stable" - 2?